products/rabbitmq.md

```diff
@@ -8,7 +8,8 @@ releasePolicyLink: https://www.rabbitmq.com/versions.html
 changelogTemplate: https://github.com/rabbitmq/rabbitmq-server/releases/tag/v__LATEST__
 releaseDateColumn: true
 activeSupportColumn: Community Support
-eolColumn: Extended Support
+eolColumn: false
```

Comment:
Up to now I think we never configured a product like that. If there is only a single support phase we are using the `eol` column for the end of support, and drop the `activeSupportColumn`. This has the advantage of having the `eol` column filled for nearly all products.

Shouldn't we keep doing that ?

Reply:
This is a good point. I would also prefer to use the `eol` instead of the `activeSupportColumn`.

Reply:
Community Support = Regular patch releases, with active development, hence I thought of using `support` instead of `eol`. Should I mark both dates `support = eol` instead?

Reply:
That would give a false impression that the commercial support is available to everyone. I want to keep the `eol = what most people would agree with` definition.

I changed support back to eol in b1ddc6e8